docs/java-rest/high-level/ml/get-buckets.asciidoc

```diff
@@ -70,7 +70,7 @@ include-tagged::{doc-tests}/MlClientDocumentationIT.java[x-pack-ml-get-buckets-s
 
 ["source","java",subs="attributes,callouts,macros"]
 --------------------------------------------------
-include-tagged::{doc-tests}/MlClientDocumentationIT.java[x-pack-ml-get-buckets-end]
+include-tagged::{doc-tests}/MlClientDocumentationIT.java[x-pack-ml-get-buckets-start]
```

Comment:
I guess it's not really part of this PR, and it's correct that the comment below applies to start rather than end, but there is no reference in the docs to the block tagged `x-pack-ml-get-buckets-end` in the test file.

Reply:
I'd rather fix it in this one. I'll add a commit.

Reply:
Actually, just noticed it is referred. You might have missed it as they are listed alphabetically.